diff --git a/CMakeLists.txt b/CMakeLists.txt
index f7e0fbd..19073a7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -246,11 +246,10 @@ if (SQLITECPP_INTERNAL_SQLITE)
     target_include_directories(sqlite3 PUBLIC "${PROJECT_SOURCE_DIR}/sqlite3")
     target_include_directories(SQLiteCpp PRIVATE "${PROJECT_SOURCE_DIR}/sqlite3")
 else (SQLITECPP_INTERNAL_SQLITE)
-    find_package (SQLite3 REQUIRED)
-    if (SQLITE3_FOUND)
-        include_directories(${SQLITE3_INCLUDE_DIRS})
-        target_link_libraries (SQLiteCpp ${SQLITE3_LIBRARIES})
-    endif (SQLITE3_FOUND)
+    message(STATUS "CMAKE_SOURCE_DIR = ${CMAKE_SOURCE_DIR}")
+    message(STATUS "CONAN_LIBS = ${CONAN_LIBS}")
+    target_include_directories(SQLiteCpp PRIVATE ${CMAKE_SOURCE_DIR})
+    target_link_libraries(SQLiteCpp PUBLIC ${CONAN_LIBS})
 endif (SQLITECPP_INTERNAL_SQLITE)
 
 # Optional additional targets: